src/vl-cookie-consent.js

```diff
@@ -117,18 +36,16 @@ export class VlCookieConsent extends VlElement(HTMLElement) {
         super(`
             <style>
                 @import '/node_modules/vl-ui-button/dist/style.css';
-                @import '/node_modules/vl-ui-checkbox/dist/style.css';
-                @import '/node_modules/vl-ui-form-message/dist/style.css';
                 @import '/node_modules/vl-ui-form-grid/dist/style.css';
                 @import '/node_modules/vl-ui-modal/dist/style.css';
             </style>
 
             <vl-modal data-title="Cookie-toestemming" not-cancellable>
                 <div is="vl-form-grid" is-stacked slot="content">
-                    <div is="vl-form-column">Het Departement Omgeving maakt op de websites waarvoor zij verantwoordelijk is gebruik van "cookies" en vergelijkbare internettechnieken. Cookies zijn kleine "tekstbestanden" die worden gebruikt om onze websites en apps beter te laten werken en jouw surfervaring te verbeteren. Zij kunnen worden opgeslagen in de context van de webbrowser(s) die je gebruikt bij het bezoeken van onze website(s).</div>
+                    <div is="vl-form-column"><span id="owner">Het Departement Omgeving</span> maakt op de websites waarvoor zij verantwoordelijk is gebruik van "cookies" en vergelijkbare internettechnieken. Cookies zijn kleine "tekstbestanden" die worden gebruikt om onze websites en apps beter te laten werken en jouw surfervaring te verbeteren. Zij kunnen worden opgeslagen in de context van de webbrowser(s) die je gebruikt bij het bezoeken van onze website(s).</div>
```

Comment:
Op de demo-pagina (https://webcomponenten.omgeving.vlaanderen.be/demo/vl-cookie-consent.html) zie ik 3x de woorden "Departement Omgeving" staan, en hier wordt er maar 1 replaced?

Reply:
Of is dat nog oud?

Reply:
Inderdaad, had niet gezien dat de eigenaar 3 keer vermeld werd. Ik pas het aan.